common/persistence/visibility/store/sql/query_converter.go

```diff
@@ -504,16 +510,22 @@ func (c *QueryConverter) convertColName(exprRef *sqlparser.Expr) (*saColName, er
 
 func (c *QueryConverter) convertValueExpr(
 	exprRef *sqlparser.Expr,
-	saName string,
+	saAlias string,
+	saFieldName string,
 	saType enumspb.IndexedValueType,
 ) error {
 	expr := *exprRef
 	switch e := expr.(type) {
 	case *sqlparser.SQLVal:
-		value, err := c.parseSQLVal(e, saName, saType)
+		value, err := c.parseSQLVal(e, saAlias, saType)
```

Comment:
Why change this?

Reply:
I thought `alias` was more descriptive than a more general `name`

Reply:
Agree, but before this change, `saName` corresponds to the field name, and you're changing to the alias. I checked the function, and it seems that it doesn't really matter at this moment if it's the alias or field name since the special parsing is only for the system search attributes, or based on the type.